Formula/neovim.rb

```diff
@@ -62,9 +62,9 @@ class Neovim < Formula
       sha256 "afda64ca445203bb7092372b822bae8b2539fdcebbfc3f753f393628c2bcfe7d"
     end
 
-    resource "luajit" do
-      url "http://luajit.org/download/LuaJIT-2.0.4.tar.gz"
-      sha256 "620fa4eb12375021bef6e4f237cbd2dd5d49e56beb414bee052c746beef1807d"
+    resource "luajit-github" do
+      url "https://github.com/LuaJIT/LuaJIT/archive/v2.0.4.tar.gz"
+      sha256 "d2abdf16bd3556c41c0aaedad76b6c227ca667be8350111d037a4c54fd43abad"
```

Comment:
@justinmk I'm guessing because this is a github generated tarball from a tag, as opposed to whatever release process the LuaJIT project uses. It does build just fine though.

I will look at the two tarballs in more detail and tell you what the exact differences are.